src/framework/ui/overflowMenu/overflowMenu.component.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {
+  ViewProps,
+  View,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+  styled,
+} from '@kitten/theme';
+import {
+  OverflowMenuItemType,
+  OverflowMenuItem as OverflowMenuItemComponent,
+  Props as OverflowMenuItemProps,
+} from './overflowMenuItem.component';
+import {
+  Popover as PopoverComponent,
+  Props as PopoverProps,
+} from '../popover/popover.component';
+import { Omit } from '../service/type';
+
+type MenuItemElement = React.ReactElement<OverflowMenuItemProps>;
+export type MenuItemType = Omit<OverflowMenuItemType, 'size'>;
+
+interface OverflowMenuProps {
+  children: React.ReactElement<any>;
+  items: MenuItemType[];
+  size?: string;
+  onSelect?: (index: number) => void;
+}
+
+const Popover = styled<PopoverComponent, PopoverProps>(PopoverComponent);
+const OverflowMenuItem =
+  styled<OverflowMenuItemComponent, OverflowMenuItemProps>(OverflowMenuItemComponent);
+
+export type Props = & StyledComponentProps & OverflowMenuProps & Omit<PopoverProps, 'content'>;
+
+export class OverflowMenu extends React.Component<Props> {
+
+  static defaultProps: Partial<Props> = {
+    size: 'medium',
+  };
+
+  private onSelect = (index: number): void => {
+    if (this.props.onSelect && this.props.visible) {
+      this.props.onSelect(index);
+    }
+  };
+
+  private isFirstItem = (index: number): boolean => {
+    return index === 0;
+  };
+
+  private isLastItem = (index: number): boolean => {
+    return this.props.items.length - 1 === index;
+  };
+
+  private isSingleItem = (): boolean => {
+    return this.props.items.length === 1;
+  };
+
+  private getPopoverStyle = (style: StyleType): StyleType => ({
+    ...style.popover,
+    borderRadius: style.borderRadius,
+  });
+
+  private getMenuItemStyle = (style: StyleType, index: number): StyleType => {
+    const borderRadius: number = style.menuItem.borderRadius;
+
+    if (this.isFirstItem(index) && !this.isSingleItem()) {
+      return {
+        borderTopLeftRadius: borderRadius,
+        borderTopRightRadius: borderRadius,
+      };
+    } else if (this.isLastItem(index) && !this.isSingleItem()) {
+      return {
+        borderBottomLeftRadius: borderRadius,
+        borderBottomRightRadius: borderRadius,
+      };
+    } else if (this.isSingleItem()) {
+      return {
+        borderRadius: borderRadius,
+      };
+    } else {
```

Comment:
final else could be omitted
just return null

Reply:
Done.